tests/wdio/reports/result-de-chrome-terra-dev-site.json

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
this should be deleted

Reply:
Removed it here https://github.com/cerner/terra-dev-site/pull/291/commits/aee2cba7f1788c819e3abb3428f716dab55a0b2c.